environment-dev.yml

```diff
@@ -5,5 +5,7 @@ dependencies:
   # Build
   - cmake
   - ninja
+  # Alternative of `std::chrono` introduced in C++20.
+  - howardhinnant_date
```

Comment:
Since this is required for LIBCPP only, I think it should not be in the general environment file.

Reply:
Yes, I wish selectors were present (see https://github.com/conda/conda/issues/8089).

Let me modify the CI accordingly.